package.json

```diff
@@ -2,7 +2,7 @@
   "name": "devkit-core",
   "author": "Game Closure",
   "description": "devkit engine",
-  "version": "v4.5.3-43",
+  "version": "v4.5.3-42",
```

Comment:
wrong, why has the version number reduced

Reply:
Oops.Mistake.